config/adopters.json

```diff
@@ -692,7 +692,16 @@
 			"logo_white": "logo-verizon-white.svg",
 			"projects": [
 				"iot.leshan"
-      ]
+      		]
+		},
+		{
```

Comment:
Please keep the JSON in alphabetic order by name, it makes management far easier

Reply:
Updated.